rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -594,8 +594,7 @@ open class InternalInterpreter(
                             it.val2,
                             value,
                             value1,
-                            value2,
-                            line
+                            value2
```

Comment:
Unfortunately, this change has caused a regression.  
Try, for example, to force an error in `MUTE13_01`, by replacing:

``` 
    MU* VAL1(AAA100) VAL2('910') COMP(EQ) 
```

with: 
``` 
    MU* VAL1(AAA100) VAL2('910') COMP(NE) 
```

If you run `MuteExecutionTest.executeMUTE13_01`, the error message doesn't provide useful information about the line. See below:

```
java.lang.AssertionError: Mute assertion failed - mute/MUTE13_01: Left value: "910       "  - right value: "910"
```